sros2/sros2/keystore/_permission.py

```diff
@@ -30,17 +34,21 @@
 )
 
 
-def create_permission(keystore_path, identity, policy_file_path):
+def create_permission(
+        keystore_path: pathlib.Path,
+        identity: str,
+        policy_file_path: pathlib.Path) -> None:
```

Comment:
```suggestion
        policy_file_path: pathlib.Path):
```

Reply:
Some context for this would be useful.

Reply:
`warn_no_return` isn't enabled by default in ament_mypy (or mypy in general), which lets mypy leverage an implicit None return type. 

With `__init__` already no longer requiring a `-> None:` it seems more consistent to use the implicit None throughout, rather than require a C style `void` annotation.